common/changes/@microsoft/rush/allow-skip-if-cache-disabled_2023-10-18-21-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Allow legacy skip behavior if build cache is configured but disabled. When running in verbose mode, log the type of incremental logic in use.",
```

Comment:
https://github.com/microsoft/rushstack-websites/pull/208 clarifies the "legacy skip" vs new behavior.

Reply:
```suggestion
      "comment": "Allow the output preservation incremental strategy if the build cache is configured but disabled. When running in verbose mode, log the incremental strategy that is being used.",
```